lib/park.rb

```diff
@@ -0,0 +1,30 @@
+class Park
+  attr_reader :name, :entrance_fee, :vehicles, :revenue
+  def initialize(name, entrance_fee)
+    @name = name
+    @entrance_fee = entrance_fee
+    @vehicles = []
+    @revenue = 0
+  end
+
+  def add_vehicle(car)
+    @vehicles << car
+  end
+
+  def park_attendees
+    @park.vehicles.each do |passenger|
+      passenger.count
+    end
+  end
+      
+
+
+  def ticket_sales
+    @vehicles.each do |person|
+      person.num_adults
+      @revenue += 50
+    end
+    @revenue
+  end
```

Comment:
```suggestion
  def ticket_sales
    @vehicles.each do |person|
      count_adults = person.num_adults
      @revenue += @admission_price * count_adults
    end
    @revenue
  end
```

This method isn't quite working. The biggest thing I notice though is that you hardcoded an admission_price of 50. We want this to dynamically calculate per park and each park has its own fee. We can make this dynamic by using the park's `admission_price` attribute. We also need to multiply that fee by the number of adults. You start that on line 24 but you'd need to set that equal to a variable so you can perform the rest of the math on line 25.